examples/stories/util.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+import * as skinview3d from "../..";
+
+export const createViewer = (config) => {
+	const width = config?.width || 300;
+	const height = config?.height || 400;
+
+	const element = document.createElement("div");
+	const viewer = new skinview3d.SkinViewer(element, {
+		width: width,
+		height: height,
+		skin: "textures/1_8_texturemap_redux.png",
+	});
+
+	const control = skinview3d.createOrbitControls(viewer);
+	control.enableRotate = true;
+	control.enableZoom = true;
+	control.enablePan = true;
+
+	return { viewer, element };
```

Comment:
It's not necessary to return `element`. Just use `domElement`.

Reply:
> If each story uses a separate SkinViewer instance, we will be unable to combine different configurations. For example, it's impossible to run when using a custom skin.
> 
> I suggest sharing a single SkinViewer instance among different stories, but I'm not sure whether it's a good practice.

I'll just make one big story that uses the same instance then as it's the best approach I think.